server/index.js

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Is this sample server still being used in this PR? It looks like we are just hitting example sites - do we still need this?

Reply:
Oh right nah I'll delete.